pkg/render/apiserver.go

```diff
@@ -1607,6 +1607,13 @@ func (c *apiServerComponent) tigeraNetworkAdminClusterRole() *rbacv1.ClusterRole
 			Verbs:         []string{"*"},
 			ResourceNames: []string{"cluster-settings", "user-settings"},
 		},
+		// Allow the user to read and write the applicationlayers resource to enable/disable WAF.
```

Comment:
I'm not sure but I think this should probably be a different role. And I'm not sure that the operator is the right place to be creating or giving access to it.
I don't think the same users that have tigera-network-admin should have access to configuring the operator and what it deploys.
It is a bit of a circular dependency to have the operator granting permissions to control the operator.

Reply:
As discussed with @tigeraphil and @malavikabala and ourselves, PM are ok for these permissions are ok to go into `tigera-network-admin` for now, possibly revisited in a future release.